File: PrevisoesClimaticas/Python/clima.py
class Clima:
    def __init__(self, ano, mes, temperatura, precipitacao):
        #constructor da classe clima, recebe o ano, mes, temperatura e precipitacao        
        self.ano = ano
        self.mes = mes
        self.temperatura = temperatura
        self.precipitacao = precipitacao

    def __str__(self):
        return f"Dados climaticos(Ano:{self.ano}, Mes:{self.mes}, Temperatura:{self.temperatura}, Precipitacao:{self.precipitacao})"

    def __eq__(self, object):
        return self.ano == object.ano and self.mes == object.mes and self.temperatura == object.temperatura and self.precipitacao == object.precipitacao

File: atividades-avaliativas/PrevisoesClimaticas/Java/Clima.java
package Java;
public class Clima{
   public String ano;
   public String mes;
   public String temperatura;
   public String precipitacao;
   
//constructor da classe clima, recebe o ano, mes, temperatura e precipitacao
    public Clima(String ano, String mes, String temperatura, String precipitacao) {
        this.ano = ano;
        this.mes = mes;
        this.temperatura = temperatura;
        this.precipitacao = precipitacao;
        
    }
    //rescrevemos o toString para imprimir os dados do clima
    @Override
    public String toString() {
        return "Clima [ano=" + this.ano + ", mes=" + this.mes + ", temperatura=" + this.temperatura + ", precipitacao=" + this.precipitacao + "]";
    }

        // rescrevemos o equals para definir os atributis identificadoes
    @Override
    public boolean equals(Object obj) {
        Clima c = (Clima) obj;
        return (this.ano.equals(c.ano) && this.mes.equals(c.mes));
    }

    

    
}